src/Mod/Sketcher/App/Sketch.cpp

```diff
@@ -234,6 +236,8 @@ int Sketch::setUpSketch(const std::vector<Part::Geometry *> &GeoList,
     Base::Console().Log("\n");
 #endif //DEBUG_BLOCK_CONSTRAINT
 
+    buildInternalAlignmentGeometryMap(ConstraintList);

```

Comment:
We don't seem to get to this line, as seen through gdb. I'm probably missing something.

Reply:
Resolving this. Looking more like an issue with `gdb`.